contributors.json

```diff
@@ -660,5 +660,12 @@
   "xxx32": {
     "country": "India",
     "name": "Aarushi"
+  },
+  "shreyaraj13": {
```

Comment:
Can you update this in ascending order alphabetically by github user name?

Reply:
okay